uni/unidoc/UniDoc.icn

```diff
@@ -167,7 +168,7 @@ procedure processArgs(uniAll # Unicon parser (instance of <tt>UniAll</tt>)
     linkPath  := []	# Default situation (no --linkPath option)
     linkPath  := [: genFields(\(Args().getOptDef("linkPath")),",") :]
     uniAll.setSourcePath(Args().getOptDef("sourcePath"))
-    uniAll.setTargetDir(targetDir := Args().getOptDef("targetDir"))
+    uniAll.setTargetDir(targetDir := Args().getOptDef("targetDir","./htmlDoc"))
```

Comment:
Since targetDir is already set to `./htmlDoc` at the top of this function I suggest just using that here.  Or maybe just delete the line since this change will always overwrite targetDir.

Reply:
I've made a slightly different change.  There is no longer a global variable targetDir and all references to the value are through UniAll's (net) getTargetDir() call.  I'll put that into the same PR as soon as I remember how to do that.  (Besides, I can't find where targetDir is preset at the top of that function!.

It looks like the build (see below) is failing again because fixName() isn't being found.  I don't see that here yet.

Reply:
Found the missing line.  I had removed it because it just gets in the way without adding any value.

Reply:
here, but it looks like  you have deleted it in the new commit
```
    local arg
    indexFile := "index.html"
    targetDir := "./htmlDoc"
```